cloudbuild/presubmit.sh

```diff
@@ -67,7 +67,7 @@ determine_tests_to_run() {
   echo "Changed directories: ${changed_dirs[*]}"
 
   # Determines what tests in changed init action directories to run
-  declare -a TESTS_TO_RUN
+  declare -g -a TESTS_TO_RUN
```

Comment:
Why put it inside of the function?

Reply:
This is how function passes result.

Do you think will be better to declare it as a global variable at the top of the script?

Reply:
Yes.

Reply:
Done